### ISSUE TITLE
"No cuda shared libraries found. Cannot continue" TTMiner KawPOW

### ISSUE THREAD

**Issue by samdave123** [RainbowMiner/RainbowMiner#1016]
TTMiner is refusing to run KawPOW on my 1070Tis today for some reason. Log file is below:

TT-Miner - Version: 5.0.2 (May 15 2020 07:52:47)

No cuda shared libraries found. Cannot continue.

**Comment by RainbowMiner** [comment]
What is the Nvidia driver version number?

**Comment by samdave123** [comment]
441.34 DCH / Win10 64

**Comment by RainbowMiner** [comment]
I am using the same. Eventually TT miner needs some extra dll. I'll check this now.

**Comment by samdave123** [comment]
It's doing it on KawPOW on my 1660Tis too now.

**Comment by samdave123** [comment]
Phoenix is saying this for my 1660Tis:

Unable to enum CUDA GPUs: unknown error
No avaiable GPUs for mining. Please check your drivers and/or hardware.

**Comment by samdave123** [comment]
I found this in gpu-count.txt

00:02.0 VGA compatible controller: Intel Corporation HD Graphics 510 (rev 06)
01:00.0 VGA compatible controller: NVIDIA Corporation GP104 [GeForce GTX 1070 Ti] (rev a1)
06:00.0 VGA compatible controller: NVIDIA Corporation TU116 [GeForce GTX 1660 Ti] (rev a1)
0a:00.0 VGA compatible controller: NVIDIA Corporation TU116 [GeForce GTX 1660 Ti] (rev a1)
0b:00.0 VGA compatible controller: NVIDIA Corporation GP104 [GeForce GTX 1070 Ti] (rev a1)
0c:00.0 VGA compatible controller: NVIDIA Corporation GP104 [GeForce GTX 1070 Ti] (rev a1)
0d:00.0 VGA compatible controller: NVIDIA Corporation GP104 [GeForce GTX 1070 Ti] (rev a1)
0e:00.0 VGA compatible controller: NVIDIA Corporation GP104 [GeForce GTX 1070 Ti] (rev a1)

This is showing 5 1070Tis and Intel graphics. I have Intel graphics but don't have any display connected. I also only have 3 1070Tis installed currently. Could this be causing me issues, and if so, how can I fix it?

**Comment by RainbowMiner** [comment]
Actually, this sounds like a driver problem. Eventually the drivers got corrupted. At first, I would try to re-install the Nvidia drivers (remember to set Force P2 State to Off, afterwards.)

**Comment by samdave123** [comment]
> 
> 
> Actually, this sounds like a driver problem. Eventually the drivers got corrupted. At first, I would try to re-install the Nvidia drivers (remember to set Force P2 State to Off, afterwards.)
> 
> The gpu-count.txt should not have any impact. It is set, when you run Install.bat

I've taken GPUs out since running Install.bat. Do I need to reconfigure RBM?

**Comment by RainbowMiner** [comment]
> I've taken GPUs out since running Install.bat. Do I need to reconfigure RBM?

No. This is only necessary, if RainbowMiner runs on a Linux machine and the rig hosts AMD GPUs.

**Comment by RainbowMiner** [comment]
Is this problem still persisting? If yes, could you eventually upload a Debug file?

**Comment by samdave123** [comment]
> 
> 
> Is this problem still persisting? If yes, could you eventually upload a Debug file?

I think it's an issue with my Asus B250 Mining Motherboard not liking my 1660Tis. I've ordered an Asrock H110 BTC+ which should hopefully resolve the issue. You might as well close this one as I'm 90% sure the issue is something to do with my motherboard.

**Comment by ArSd-g** [comment]
again it's the --opencl-plateform parameter the problem try KawPOW standalone without and it works.remove nvidia/amd palteform parameter and let auto-detect would be the solution!

**Comment by samdave123** [comment]
> 
> 
> again it's the --opencl-plateform parameter the problem try KawPOW standalone without and it works.remove nvidia/amd palteform parameter and let auto-detect would be the solution!
> i have the same error with Vega 56 (amd so...)
> remember RainbowMiner, i think it was the same issue on NBminer... ;)

This is an issue with my motherboard.

**Comment by samdave123** [comment]
I've resolved the problems with my rig, but TTminer is still acting funny. I've got my GPUs split into different device groups, but TTminer is not working when running on more than one card.

**Comment by RainbowMiner** [comment]
Is it, that two TTminers running in parallel fail? Or is it one TTminer failing to run on more than one GPU?

**Comment by samdave123** [comment]
> 
> 
> Is it, that two TTminers running in parallel fail? Or is it one TTminer failing to run on more than one GPU?

One instance runs fine, then the 3 others fail.
